### PR TITLE
Tip to get original object back from Persistent or PersistentBase

### PR DESCRIPTION
**Pull request by brodybits**
Thanks to @rvagg (#475)

Also remove trailing whitespace from doc/persistent.md

### CODE REVIEW

**Comment by rvagg** [comment]
lgtm, would prefer @kkoopa to sign off on the wording

**Comment by brodybits** [comment]
>
> lgtm, would prefer @kkoopa <https://github.com/kkoopa> to sign off on the
> wording
>

:+1:

>

-- 
Sent from my mobile

File: doc/persistent.md [hunk 1]
```diff
@@ -95,6 +95,12 @@ template<typename T> class PersistentBase {
 
 See the V8 documentation for [`PersistentBase`](https://v8docs.nodesource.com/io.js-3.0/d4/dca/classv8_1_1_persistent_base.html) for further information.
 
+**Tip:** To get the original `v8::Local` object reference back from a `PersistentBase` or `Persistent` object:
```
Comment (kkoopa) [inline doc/persistent.md]: You get a new reference to the original object, not the original reference.

**Comment by brodybits** [comment]
Wording changed, please check again thanks.

**Comment by kkoopa** [comment]
Yes, LGTM.

**Comment by kkoopa** [comment]
Merged as a0af18782adfc09ff0ae4a00a86241f504fd05c2

**Pull request closed without merging**